andela/asgard-cfh
# Ft upload profile image 157801309

#### What does this PR do?
- Implements Social Authentication for signup and also successful image upload on manual signup 

#### Description of Task to be completed?
- users should be able to upload images while signing up
- users should be able to signup using any social platform requested
- images of users on social platform should be used as default image

#### How should this be manually tested?
-  visit application [here](https://asgard-cfh-staging.herokuapp.com/)
- signup with any social platform and you would see your image use as default image on gaming screen instead of an avatar
-  select an image while signing up and your image should reflect on the gaming screen

#### Any background context you want to provide?
- 
#### What are the relevant pivotal tracker stories?
[157801309](https://www.pivotaltracker.com/story/show/157801309)

#### Screenshots (if appropriate)

#### Questions:
- none
